<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="INLTP-SHILPIM" name="testScriptsSmoke.TS_004_BasicAndAdvancedSearchOperations" tests="6" failures="2" timestamp="04 Apr 2020 09:35:57 GMT" time="789.158" errors="0">
  <testcase name="contentSearchWithOperatorsInBS" time="205.237" classname="testScriptsSmoke.TS_004_BasicAndAdvancedSearchOperations">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [3] but found [2],
	expected [11] but found [10],
	expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [3] but found [2],
	expected [11] but found [10],
	expected [2] but found [1]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testScriptsSmoke.TS_004_BasicAndAdvancedSearchOperations.contentSearchWithOperatorsInBS(TS_004_BasicAndAdvancedSearchOperations.java:94)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1297)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- contentSearchWithOperatorsInBS -->
  <testcase name="audioSearch" time="25.741" classname="testScriptsSmoke.TS_004_BasicAndAdvancedSearchOperations"/>
  <testcase name="contentSearchWithOperatorsInAS" time="190.763" classname="testScriptsSmoke.TS_004_BasicAndAdvancedSearchOperations">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [3] but found [2],
	expected [11] but found [10],
	expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [3] but found [2],
	expected [11] but found [10],
	expected [2] but found [1]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testScriptsSmoke.TS_004_BasicAndAdvancedSearchOperations.contentSearchWithOperatorsInAS(TS_004_BasicAndAdvancedSearchOperations.java:121)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1297)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- contentSearchWithOperatorsInAS -->
  <testcase name="metaDataSearch" time="271.751" classname="testScriptsSmoke.TS_004_BasicAndAdvancedSearchOperations"/>
  <testcase name="searchsavedSearch" time="70.142" classname="testScriptsSmoke.TS_004_BasicAndAdvancedSearchOperations"/>
  <testcase name="ConceptualSearch" time="25.524" classname="testScriptsSmoke.TS_004_BasicAndAdvancedSearchOperations"/>
</testsuite> <!-- testScriptsSmoke.TS_004_BasicAndAdvancedSearchOperations -->
